### PR TITLE
Add support for changing cards

### LINKED ISSUE #95: "Connect new card with user" feature in admin

### PR DESCRIPTION
**Pull request by ekmartin**
Fixes #95.
Not ready yet, needs frontend tests.

image

EDIT: Should probably also return 400 if a card key isn't given in the payload.

### CODE REVIEW

**Comment by ekmartin** [comment]
Added a few frontend tests for changing card keys now, should be good to go.

File: features/admin.feature [hunk 1]
```diff
@@ -85,3 +85,30 @@ Feature: Admin
     And I am on the edit election page
     Then I should see 1 in "votedUsers"
     And I should see 1 in "activeUsers"
+
+  Scenario: Changing the card key of a user
+    Given There is an active user with card key "1234"
+    And I am waiting on page "/admin/change_card"
+    When I scan card key "1235"
+    And I fill in model "user.username" with "testuser"
```
Comment (relekang) [inline features/admin.feature]: "I fill in model" describes functionality in the angular app not what the user does.

Comment (ekmartin) [inline features/admin.feature]: Yep, I'll change it.

**Comment by JuulArthur** [comment]
Goodie!

**Pull request merged**